langchain/retrievers/document_compressors/chain_extract.py

```diff
@@ -69,9 +69,10 @@ def from_llm(
         llm: BaseLanguageModel,
         prompt: Optional[PromptTemplate] = None,
         get_input: Optional[Callable[[str, Document], str]] = None,
+        llm_chain_kwargs: Union[Dict[str, Any], None] = None,
```

Comment:
this can just be `Optional[dict] = None`

Reply:
Thanks, sorry we are on py3.10, forgot all about `Optional` ... :)